content/utils.js

```diff
@@ -69,8 +70,24 @@ function memoize(fn) {
   };
 }
 
+function execGit(args, opts = {}) {
+  const { status, error, stdout, stderr } = childProcess.spawnSync(
+    "git",
+    args,
+    {
+      cwd: CONTENT_ROOT,
```

Comment:
Ah! I see. That's why it actually worked (when I tested it) despite my previous comments somewhere.